### DISCUSSION TITLE
How to get images processed correctly during build process

### DISCUSSION THREAD

**Discussion by l0ggik** [nuxt/image#1130] [Q&A]
Hi,

i try to understand the build process for a project with nuxt image.

I have put my images in /assets/images
In dev mode everything works perfectly fine but when build the project with npm run build the images in /assets/images are not processed and the images are missing in the production build.

So my question is how do i have to configure the build process to also process images during build?

I can't find any helpful documentation regarding this.

**Comment by desk-pro** [comment]
Same problem here. I am using Firebase Hosting

**Comment by luisjoserivera** [comment]
Resizing happens at runtime.

[Nuxt/Image Configuration - dir](https://image.nuxt.com/get-started/configuration#dir) has a note for this:

 _"For some providers (like vercel), using a directory other than public/ for assets is not supported since resizing happens at runtime (instead of build/generate time) and source fetched from the public/ directory (deployment URL)"_

This is working in dev mode because the assets folder does exist in your local environment but not in your built.
Files in the assets folder are meant to be processed and copied to the public directory, which is what nuxt-image does on prerendered pages that contains static images. But since you are deploying a build that probably is not prerendering your targeted images these images are not available in your deployment.

If the images in question belong to static pages you could go [hybrid](https://nuxt.com/docs/guide/concepts/rendering#hybrid-rendering) and prerender this pages during your build.

If the images in question belong to dynamic pages you might want to check this as well [Static Images](https://image.nuxt.com/advanced/static-images)
